### PR TITLE
try to rePR phylodiversity tuto

### PR DESCRIPTION
**Pull request by yvanlebras**
TODO: Describe your changes here..

### CODE REVIEW

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+objectives:
+- Learning how to use the phylodiversity workflow.
+- Compute endemism index
+- Create a phyloregion map
+time_estimation: ''
+tags:
+- ecology
+- taxonomic data
+contributors:
+- Morphy427
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]:
```suggestion
contributions:
  authorship:
    - Morphy427
```

(We changed slightly the way we do contributor metadata)

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+---
+
+
+# Introduction
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]:
```suggestion
```

(removing this makes the link previews a bit nicer when you share a link to the tutorial on e.g. social media or via chat)

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
+---
+layout: tutorial_hands_on
+
+title: Phylodiversity workflow quick tutorial
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: from the title I expected the hands-on to involve running a workflow. So maybe replace "workflow" with "analysis" orso?

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+>    >
+>    > The output of this tool is a shapefile withe the clusterisation done in function of the endemism. You'll have to choose a number of cluster you want and the clustering method you want.
+>    {: .comment}
+>    > <comment-title> more tips and info </comment-title>
+>    >
+>    > If you have no idea how many cluster you want, the tool start with an estimation of how many clusters are optimal between 0 to 30. So you can firstly run the tool with default value and go check the standard output to check the recommanded number. However keep in mind that this estimation is purely statistics and don't always have biologic reasons.
+>    {: .comment}
+>
+{: .hands_on}
+{: .hands_on}
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: double hands on closing here
```suggestion
```

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+>
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets
+> 4. Check that the datatype of the phylogenic phyle is newick, the occupancy file is tabular and the shape file is a composite dataset of type shapefile
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]:
```suggestion
>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="newick" %}
```

In the snippet you can pass a parameter for the datatype you want people to select

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    ```
+>    https://zenodo.org/api/records/15487657/files/shapefile.dbf/content
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: we can rewrite the Zenodo links a bit:

```suggestion
>    https://zenodo.org/records/15487657/files/shapefile.dbf
```
(and do the same for other links)

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+>
+> 5. Add to each database a tag corresponding to ...
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+# Data formatting
+
+The first step is to be sure your data are well formated. If all your file are in good format and do have the needed column as specified before, you can move forward.
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md:117]: maybe add the first few lines of the file here so that people can compare their file to what it should look like?

**Review by shiltemann — COMMENTED** [review]
Thanks a lot @Morphy427 and @yvanlebras!

Could you add @Morphy427 to the CONTRIBITORS.yaml file?

few small technical comments/suggestions below

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md (hunk 6 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: yes, thank you!

**Comment by yvanlebras** [comment]
ok, I think I found the ""last ??"" ;) issue and made a PR. Don't hesitate if you prefer to give me write access to your training-material repo to avoid the need to accept PR ;)

**Comment by shiltemann** [comment]
thanks @yvanlebras! Do you want to assign the tutorial to one of the subtopics? 

If you removed the workflow because of missing workflow tests, we can also merge and you add the workflow tests later (better to have a workflow without tests than none at all).

but otherwise this is good from my side

**Review by shiltemann — APPROVED** [review]

**Comment by yvanlebras** [comment]
Almost ok on my side. It seems to me @Morphy427 the https://zenodo.org/records/15487657/files/grid_test.csv datafile is not still perfect... This file is a real nightmare ;) Elouan, an attempt to propose a corrected filee here: https://ecology.usegalaxy.eu/api/datasets/4838ba20a6d86765fb45f5f51ef18946/display?to_ext=tabular if it seems ok for you, you can use it to update the Zenodo file.

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md (hunk 7 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md:117]: I tried to add a screenshot of an example data file

**Review by yvanlebras — COMMENTED** [review]
seems to be ok!

**Comment by yvanlebras** [comment]
> @Morphy427 if you updated the file on Zenodo, please also update the Zenodo URLs in the tutorial, and data-library.yaml file

I am on it! Thank you Saskia!

**Comment by yvanlebras** [comment]
ok, all seems to be ok on my side !
You can review and merge Saskia if ok for you!

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
+---
+layout: tutorial_hands_on
+
+title: Phylodiversity analysis quick tutorial
+zenodo_link: https://doi.org/10.5281/zenodo.15487657
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]:
```suggestion
zenodo_link: https://zenodo.org/records/15601932
```

File: topics/ecology/tutorials/phylodiversity_workflow/data-library.yaml [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+        ext: ''
+        info: https://doi.org/10.5281/zenodo.15487657
+      - url: https://zenodo.org/api/records/15487657/files/composite_dataset_shp.html/content
+        src: url
+        ext: ''
+        info: https://doi.org/10.5281/zenodo.15487657
+      - url: https://zenodo.org/api/records/15487657/files/grid_test.csv/content
+        src: url
+        ext: ''
+        info: https://doi.org/10.5281/zenodo.15487657
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/data-library.yaml]:
```suggestion
---
destination:
  type: library
  name: GTN - Material
  description: Galaxy Training Network Material
  synopsis: Galaxy Training Network Material. See https://training.galaxyproject.org
items:
- name: Ecology
  description: Learn to analyse Ecological data through Galaxy.
  items:
  - name: Phylodiversity analysis quick tutorial
    items:
    - name: 'DOI: 10.5281/zenodo.15601932'
      description: latest
      items:
      - url: https://zenodo.org/api/records/15601932/files/shapefile.shx/content
        src: url
        ext: auto
        info: https://zenodo.org/records/15601932
      - url: https://zenodo.org/api/records/15601932/files/shapefile.shp/content
        src: url
        ext: auto
        info: https://zenodo.org/records/15601932
      - url: https://zenodo.org/api/records/15601932/files/composite_dataset_shp.html/content
        src: url
        ext: auto
        info: https://zenodo.org/records/15601932
      - url: https://zenodo.org/api/records/15601932/files/grid_test.tabular/content
        src: url
        ext: auto
        info: https://zenodo.org/records/15601932
      - url: https://zenodo.org/api/records/15601932/files/shapefile.dbf/content
        src: url
        ext: auto
        info: https://zenodo.org/records/15601932
      - url: https://zenodo.org/api/records/15601932/files/shapefile.prj/content
        src: url
        ext: auto
        info: https://zenodo.org/records/15601932
      - url: https://zenodo.org/api/records/15601932/files/phylogeny_test/content
        src: url
        ext: auto
        info: https://zenodo.org/records/15601932
```

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md (hunk 8 above)
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]:
```suggestion
zenodo_link: https://zenodo.org/records/15601932
```

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+- Create a phyloregion map
+time_estimation: ''
+tags:
+- ecology
+- taxonomic data
+contributions:
+  authorship:
+    - Morphy427
+    - yvanlebras
+
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md:24]: 1. could you add some `key_points`?
2. do you want to add a `sub_topic:`?

**Comment by shiltemann** [comment]
I am not allowed to push to your branch, so I have made code suggestions for the remaining Zenodo updates, and just a small comment for some missing metadata. Thanks both!

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md (hunk 10 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md:24]: oh yes, you're right ! I am on it! THANK you Saskia

Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md:24]: done

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md (hunk 8 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: THANK YOU Saskia !

File: topics/ecology/tutorials/phylodiversity_workflow/data-library.yaml (hunk 9 above)
Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/data-library.yaml]: Oh THANK  YOU !! Sorry to have missed this data library file....

Comment (shiltemann) [reply topics/ecology/tutorials/phylodiversity_workflow/data-library.yaml]: This still needs to be applied (but I don't have permission to)

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md (hunk 8 above)
Comment (shiltemann) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: This also still needs to be done

File: topics/ecology/tutorials/phylodiversity_workflow/tutorial.md [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+- How to construct phyloregions from occurrences species data, phylogenic data and geograpics data?
+objectives:
+- Learning how to use the phylodiversity workflow.
+- Compute endemism index
+- Create a phyloregion map
+time_estimation: '2H'
+key_points:
+- Compute Phylodiversity metrics
+- Estimate the endemism of spatial units from phylogeny and occupancy data
+- Use an Essential Biodiversity Variables workflow
```
Comment (shiltemann) [inline topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: sorry to be picky, but theses arent realy key point, they are objectives. Key points should be more like "if there are 3 lessons students should have learned from this tutorial, what are they". 

So more like "When doing X, it is always important to remember..", "Tool Y is useful for doing..", "Performing step Z is important because.."

Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: you are right! Thank you!! I am on it!

Comment (yvanlebras) [reply topics/ecology/tutorials/phylodiversity_workflow/tutorial.md]: ok.. I made a first try, but it seems to me we need to propose a complete new review phasis by an expert of the domain so we will propose a new set of amelioration, notably on these key points and related explanations on the turorial. If it seems ok for you to merge it! Thank you so much !!

**Comment by shiltemann** [comment]
Thanks @yvanlebras, if you could just apply the two code suggestions I made (the Zenodo link and data library.yaml file) then we can merge. (or give me write access to the repo then I can push them myself)

**Review by shiltemann — APPROVED** [review]
